### ISSUE TITLE
Ability to check for new versions

### ISSUE THREAD

**Issue by maxdeepfield** [Eoinocal/Halite#10]
With optional configurable update auto-reminder.

**Comment by Eoinocal** [comment]
I'm not discounting this suggestion, but personally I would prefer to support Halite updates through chocolatey.org.

**Comment by maxdeepfield** [comment]
I mean make simple option "Check for new version" somewhere in user interface, maybe in Settings dialog, but this requires some cloud/server response and/or links to new version download page. Page can be chocolatey or github or whatever.

**Comment by apexlee** [comment]
I agree with maxdeepfield. Some sort of update notification built into the program would be great.
